### DISCUSSION TITLE
https redirect?

### DISCUSSION THREAD

**Discussion by jasheal** [facebook/docusaurus#3939] [Q&A] [answered]
Hello,

How do you do a server redirect `http:` ~> `https` in docusaurus(v2) ?

J

**Comment by slorber** [comment, marked as answer]
Hi

You don't.

Docusaurus only creates a build folder with HTML files, and has no way to control how these HTML files are served.
The server-side redirects all have to be configured on your hosting provider.

**Reply by slorber** [reply]
(you can add a script to do a redirect with JS, but it's always better to handle redirects on your hosting provider)

**Reply by jasheal** [reply]
Thanks @slorber - so there's no way to customize the `docusaurus serve` command to allow for a custom redirect? I can always provide my own express server I guess.

**Reply by slorber** [reply]
Docusaurus serve is not meant to be used for production, it's just a tiny util we provide to test a production build locally.
It is just a wrapper around https://github.com/vercel/serve

You are free to customize that code yourself with nodejs code for anything more advanced, or use any other solution to serve a static folder locally, including apache, nging, varnish...


I'd advise you to a modern jamstack host like Netlify, and create a `static/_redirects` file. 
That's what we do for Docusaurus site itself.
https://docs.netlify.com/routing/redirects/
